packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -167,6 +182,48 @@ contract L2ToL2CrossDomainMessengerTest is Test {
         });
     }
 
+    function testFuzz_relayMessage2_succeeds(
+        uint256 _source,
+        uint256 _nonce,
+        address _sender,
+        address _target,
+        bytes calldata _message,
+        uint256 _value
+    )
+        external
+    {
+        // Ensure that the target contract is not CrossL2Inbox or L2ToL2CrossDomainMessenger
+        vm.assume(_target != Predeploys.CROSS_L2_INBOX && _target != Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER);
+
+        // Ensure that the target call is payable if value is sent
+        if (_value > 0) assumePayable(_target);
+
+        // Ensure that the target contract does not revert
+        vm.mockCall({ callee: _target, msgValue: _value, data: _message, returnData: abi.encode(true) });
+
+        // construct the payload & identifier
+        bytes32 sig = keccak256("SentMessage(uint256,address,uint256,address,bytes)");
```

Comment:
You can use the `.selector` property on the event definition, so `L2ToL2CrossDomainMessenger.SentMessage.selector` should work here